documentation/modules/vmware-privileges.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="vmware-privileges_{context}"]
+= VMware privileges
+
+The following VMware privileges are required to migrate virtual machines to {virt} with the {project-first}.
+
+[cols="2", options="header"]
+.VMware privileges
+|===
+|Privilege |Description
+|`Virtual machine.Interaction` privileges:|
```

Comment:
@sbeskin-redhat I don't understand this line. It appears to be a lead-in sentence, but it's the first cell without a corresponding Description.